ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/block/BlockProcessor.java

```diff
@@ -123,4 +123,6 @@ void processExecutionPayload(
       ExecutionPayload executionPayload,
       OptimisticExecutionPayloadExecutor payloadExecutor)
       throws BlockProcessingException;
+
+  boolean isOptimistic();
```

Comment:
could potentially default this, not fussed either way since it's only a couple of implementations I guess...

Reply:
Yeah I deliberately didn't default it because it's very unlikely the default answer is going to be right for new implementations and I don't want them to forget to override it.  The only secure option is to default it to `true` which means overriding in most places.